privaterelay/templates/profile.html

```diff
@@ -87,8 +87,21 @@
         <span class="dashboard-greeting-message">{% ftlmsg 'profile-label-welcome-html' email=request.user.email %}
       </div>
       {% endif %}
-      {% include "includes/banners.html" %}
 
+      <!-- Data Notification Banner -->
+      {% include "includes/banners/announcement-banner.html" %}
```

Comment:
All 'permanently at the top' announcement banners can be edited here. I placed the data notification module in here.